ui/types/shim-lang.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module "quasar/lang/*" {
+  // We know "quasar" will exists at runtime, we can safely ignore the TS error
+  // @ts-ignore
```

Comment:
I think it would be better to have here `@ts-expect-error` if we really know that error is expected

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error

No, as `@ts-expect-error` will throw when the error isn't thrown, that's not what we want